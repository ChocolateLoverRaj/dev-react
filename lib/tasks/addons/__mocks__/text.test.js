import TextAddon, { _reset as resetTextAddon } from './text.js'
import Addon from './addon.js'

afterEach(resetTextAddon)

test('extends addon', () => {
  const addon = new TextAddon()
  expect(addon).toBeInstanceOf(Addon)
})

test('text property', () => {
  expect(new TextAddon().text).toMatchSnapshot()
})

test('_reset', () => {
  const addon = new TextAddon()
  addon.getText()
  resetTextAddon()
  expect(TextAddon.prototype.getText.notCalled).toBe(true)
})
