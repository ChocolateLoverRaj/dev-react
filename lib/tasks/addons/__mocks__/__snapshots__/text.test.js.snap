// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text property 1`] = `"text"`;
